src/mbgl/text/tagged_string.hpp

```diff
@@ -0,0 +1,83 @@
+#pragma once
+
+#include <mbgl/text/glyph.hpp>
+
+namespace mbgl {
+
+using StyledText = std::pair<std::u16string, std::vector<uint8_t>>;
+    
+struct SectionOptions {
+    SectionOptions(double scale_, FontStackHash fontStackHash_)
+        : scale(scale_), fontStackHash(fontStackHash_)
+    {}
+    
+    double scale;
+    FontStackHash fontStackHash;
+};
+
+struct TaggedString {
```

Comment:
Comment would be useful to indicate that the `SectionOptions` and `TaggedString` structs represent evaluated values.